### ISSUE TITLE
Create an F-Droid repository

### ISSUE THREAD

**Issue by 12people** [bravepipeproject/BravePipeExtractor#64]
I would like to get updates via F-Droid, just like I can with NewPipe: https://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501 .

This is not a request for inclusion in F-Droid, but rather for the creation of a repository I could manually add to F-Droid so that I could get updates from there.

**Comment by ghost** [comment]
Agreed, my ass always forgets about updates and I don't want to be 20 major releases ahead once I figure out I forgot to update it

**Comment by zeorin** [comment]
https://apt.izzysoft.de/fdroid/index/apk/org.polymorphicshade.newpipe
